### DISCUSSION TITLE
Add a character from the current match in coc#pum menu

### DISCUSSION THREAD

**Discussion by Festiu** [neoclide/coc.nvim#4309] [Q&A] [answered]
Hello, I'd like to know if there is a way to add one character from the current match in the new coc#pum menu. So to speak, to replicate the behaviour that has CTRL-L on the Vim's native pum menu

**Comment by chemzqm** [comment]
Not yet.

**Comment by chemzqm** [comment, marked as answer]
Fixed by d3f1a5cb

**Reply by Festiu** [reply]
Wow, thank you sir!
